learning_resources/models.py

```diff
@@ -233,7 +233,9 @@ class LearningResourceRun(TimestampedModel):
     image = models.ForeignKey(
         LearningResourceImage, null=True, blank=True, on_delete=models.deletion.SET_NULL
     )
-    level = models.CharField(max_length=128, null=True, blank=True)  # noqa: DJ001
+    level = ArrayField(
```

Comment:
I thought about changing this field name to the plural `levels` but ultimately decided to keep it as is.  I can be convinced otherwise though.